app/nhs-health-check-online/2025/11/managing-live-updates/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Designing for change: how we’re managing live updates in our service"
+date: 2025-11-01
+---
+
+The NHS Digital Health Check online service is currently in its private beta phase. It is at this stage we’re slowly beginning to observe how real users interact with our platform, and how the algorithms we use are received. This is a critical moment for any digital health product: the transition from static design to dynamic, real-world use. One of the most important challenges we’ve tackled during this phase is how to ‘manage change once live’ - especially when those changes affect users mid-journey.
+
+## The context: evolving risk algorithms 
+
+Our service uses validated algorithms like the General Practice Assessment Questionnaire (GPAQ) and QRisk3 to calculate cardiovascular risk scores and identify key health risk factors. These tools are central to our user experience; they help translate complex clinical data into actionable insights for users.
```

Comment:
"actionable insights" feels quite jargony – can you say something clearer like "information they can use to improve their health"?